wemake_python_styleguide/visitors/ast/exceptions.py

```diff
@@ -66,53 +67,23 @@ def visit_Try(self, node: ast.Try) -> None:
             UselessFinallyViolation
             DuplicateExceptionViolation
             TryExceptMultipleReturnPathViolation
+            IncorrectExceptOrderViolation
 
         """
         self._check_if_needs_except(node)
         self._check_duplicate_exceptions(node)
         self._check_return_path(node)
-        self.generic_visit(node)
-
-    def visit_ExceptHandler(self, node: ast.ExceptHandler) -> None:
-        """
-        Checks all ``ExceptionHandler`` nodes.
-
-        Raises:
-            BaseExceptionViolation
-            UselessExceptCaseViolation
-
-        """
-        self._check_useless_except(node)
-        self._check_exception_type(node)
+        self._check_exception_order(node)
         self.generic_visit(node)
 
     def _check_if_needs_except(self, node: ast.Try) -> None:
         if node.finalbody and not node.handlers:
             self.add_violation(UselessFinallyViolation(node))
 
-    def _check_exception_type(self, node: ast.ExceptHandler) -> None:
-        exception_name = node.type
-        if exception_name is None:
-            return
-
-        exception_id = getattr(exception_name, 'id', None)
-        if exception_id == self._base_exception:
-            self.add_violation(BaseExceptionViolation(node))
-
     def _check_duplicate_exceptions(self, node: ast.Try) -> None:
-        exceptions: List[str] = []
-        for exc_handler in node.handlers:
-            # There might be complex things hidden inside an exception type,
-            # so we want to get the string representation of it:
-            if isinstance(exc_handler.type, ast.Name):
-                exceptions.append(source.node_to_string(exc_handler.type))
-            elif isinstance(exc_handler.type, ast.Tuple):
-                exceptions.extend([
-                    source.node_to_string(node)
-                    for node in exc_handler.type.elts
-                ])
-
-        for exc_name, count in Counter(exceptions).items():
+        exceptions_list: List[str] = exceptions.get_all_exception_names(node)
```

Comment:
Why do we need an annotation here? Is the `exceptions.get_all_exception_names` return type is not enough?

Reply:
I think I just forgot to change/remove these two lines. Thanks for pointing that out. I will submit a fix now.